### PR TITLE
Development1

### PR DESCRIPTION
**Pull request by onedebos**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,575 @@
...
+
+<body>
+    <nav class="nav-bar">
+        <i class="fas fa-bars"></i>
+        <i class="fas fa-search"></i>
+        <a href="#" class="space-text">SPACE & COSMOS</a>
+        <img class="nyt-heading" src="assets/svg-icons/svg2.svg">
+        <button class="subscribe-btn">SUBSCRIBE NOW</button>
+        <button class="log-in-btn">LOG IN</button>
+    </nav>
```
Comment (bolah2009) [inline index.html:27]: `AWESOME`

Great work using a `<nav>` tag for navigation bar, this helps to communicate meaning to the browser, developer, reader, and any other technologies interpreting the document (e.g. voice assistants). :ok_hand:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,575 @@
...
+    <script src="https://kit.fontawesome.com/e6d0becbe7.js"></script>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <nav class="nav-bar">
+        <i class="fas fa-bars"></i>
+        <i class="fas fa-search"></i>
+        <a href="#" class="space-text">SPACE & COSMOS</a>
+        <img class="nyt-heading" src="assets/svg-icons/svg2.svg">
```
Comment (bolah2009) [inline index.html]: `REQUIRED`

`The NY Times logo is a link` - The `` element should be wrapped with a link.

-----


`SUGGESTION`

Consider adding an `alt` attribute to the `` element, this is a very important accessibility attribute.
`Examples of scenarios where users benefit from text alternatives for images` are listed below and could also be found [here](https://www.w3.org/TR/html51/semantics-embedded-content.html#alt-text)

- They have a very slow connection and are browsing with images disabled.
- They have a vision impairment and use text to speech software.
- They have a cognitive impairment and use text to speech software.
- They are using a text-only browser.
- They are listening to the page being read out by a voice Web browser.
- They have images disabled to save on download costs.
- They have problems loading images or the source of an image is wrong.


##### SOURCE: [W3C Recommendation (Embedded Content)](https://www.w3.org/TR/html51/semantics-embedded-content.html#alt-text)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,575 @@
+<!DOCTYPE html>
+
+<head>
```
Comment (bolah2009) [inline index.html:3]: `SUGGESTION`

Consider adding `lang="en"` (or variant) to the `html` start tag, this is good for internationalisation. Also, check this [link for more info](https://www.w3.org/International/questions/qa-html-language-declarations#basics)

Take advantage of [HTML CHECKER](https://validator.w3.org) by checking your code with this online tool. It spots out common mistakes/omission in HTML codes. :+1:

File: index.html [hunk 4]
```diff
@@ -0,0 +1,575 @@
...
+                Dr. Kovac and his collaborators, working in an experiment known as Bicep, for Background Imaging of
+                Cosmic Extragalactic Polarization, reported their results in a scientific briefing at the Center for
+                Astrophysics here on Monday and in a set of papers submitted to The Astrophysical Journal.
+
+            </p>
+        </div>
+               
+    </div>
+
+        </section>
```
Comment (bolah2009) [inline index.html]: Please see [#1 Comment](https://github.com/onedebos/nyt-clone/pull/6#discussion_r303197464) on [HTML Checker](https://validator.w3.org/)

File: index.html [hunk 5]
```diff
@@ -0,0 +1,575 @@
...
+        <section class="inflation-theory-section">
+                <div class="text-on-img">
+                    <h1 class="inflation-theory">The Theory of Inflation</h1>
+                    <p>Astronomers have found evidence to support the theory of inflation, which explains how the universe
+                    expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.</p>
+                <img src="assets/inflation-945.png" class="inflation-img">
```
Comment (bolah2009) [inline index.html]: Please see [#1 Comment](https://github.com/onedebos/nyt-clone/pull/6#discussion_r303197464) on [HTML Checker](https://validator.w3.org/) and [#2 Comment](https://github.com/onedebos/nyt-clone/pull/6#discussion_r303196769) on `` tag `alt`

File: index.html [hunk 6]
```diff
@@ -0,0 +1,575 @@
...
+    
+                <div>
+    
+                    <p class="p-class" id="p7"><span class="content-title">FLUCTUATION</span> Astronomers have now detected evidence of these ancient fluctuations in swirls of polarized light in the
+                    cosmic background radiation, which is energy left over from the early universe. These are gravitational
+                    waves predicted by Einstein.</p>
+                </div>
+    
+                <div>
+                    <p class="p-class" id="p8"><span">EXPANSION</span> Returning to our coffee, imagine a single, central pot expanding faster than light and cooling to an even
```
Comment (bolah2009) [inline index.html]: Please see [#1 Comment](https://github.com/onedebos/nyt-clone/pull/6#discussion_r303197464) on [HTML Checker](https://validator.w3.org/)

File: index.html [hunk 7]
```diff
@@ -0,0 +1,575 @@
...
+                                What to Expect When You’re Expecting Evil
+                            </p>
+                        </div>
+                        <div class="aside-editors-picks-section">
+                            <img src="assets/editors-picks/editor-img2.jpg" alt="ec publications" />
+                            <p class="aside-editors-caption">
+                                Drink a Pint, Waste Less Food
+                            </p>
+                        </div>
+                </section> -->
```
Comment (bolah2009) [inline index.html]: Consider deleting commented out code, If you need to review a previous version of your code, use [git history](https://git-scm.com/book/en/v2/Git-Basics-Viewing-the-Commit-History).

File: index.html [hunk 8]
```diff
@@ -0,0 +1,575 @@
...
+    <nav class="nav-bar">
+        <i class="fas fa-bars"></i>
+        <i class="fas fa-search"></i>
+        <a href="#" class="space-text">SPACE & COSMOS</a>
+        <img class="nyt-heading" src="assets/svg-icons/svg2.svg">
+        <button class="subscribe-btn">SUBSCRIBE NOW</button>
+        <button class="log-in-btn">LOG IN</button>
+    </nav>
+    <hr class="nav-hr" />
+    <main>
```
Comment (bolah2009) [inline index.html:29]: `AWESOME` 

Very good, using `main`, this shows the dominant content of the `<body>` of a document. More on `<main>` [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main) :star2: :star:

**Comment by Ispirett** [comment]
Hey folks great work so far on this project :100: There are some issues that need addressing.
- [x] Use this https://validator.w3.org tool and  correct all the HTML errors 
- [x] Some of the svgs are very big they are taking up a lot of the page please address this
- [x]  Make the section under the Post image centered properly
- [x] For the  'Read comments' button make sure its style and positioned properly 
- [x] For the social icons next to the read comments button **Microverse** recommends the use of font awesome for icons link here https://fontawesome.com


When you are finished with changes make another pull request you can use this same link

File: indexFILE.txt [hunk 9]
```diff
@@ -0,0 +1,432 @@
+<!DOCTYPE html>
```
Comment (bolah2009) [inline indexFILE.txt]: I would suggest deleting this file. :thinking:

File: CSSFile.txt [hunk 10]
```diff
@@ -0,0 +1,66 @@
+*,::before,::after{
```
Comment (bolah2009) [inline CSSFile.txt]: I would suggest deleting this file. :thinking:

File: styles.css [hunk 11]
```diff
@@ -0,0 +1,322 @@
...
+
+.main-article {
+    margin:0 auto;
+    /* display: block; */
+    width:960px;
+}
+.main-article-icons{
+    display: flex;
+    
+/*    justify-content: space-between;
```
Comment (bolah2009) [inline styles.css]: Consider deleting commented out codes.

File: styles.css [hunk 12]
```diff
@@ -0,0 +1,322 @@
...
+    width:20px;
+    height:20px;
+    background-color:#eee;
+    border:1px solid #eee;
+    padding:2px;
+    border-radius:100%;
+}
+
+.nav-bar{
+    display: flex;
```
Comment (bolah2009) [inline styles.css:50]: Consider adding `position: fixed;` to make the nav-bar fixed as required. This feature is useful with pages with long height, it makes it easier to navigate the page anywhere you are on the site.

**Review by bolah2009 — CHANGES_REQUESTED** [review]
## Great Work Team! :clap:

This was an excellent effort!  :ok_hand: 
You are almost there! Only a few changes required. :blush:

To make the review even much easier for you, below are the checklist to consider the project 'complete' it must satisfy the following points:

#### Navigation bar

:heavy_check_mark:   It has a navigation bar

:heavy_check_mark:   It uses the `<nav>` tag

:heavy_check_mark:   It has a "Hamburger button", a search icon and a link with text on the leftmost side of the navbar

:heavy_check_mark:   has the NY Times image logo embedded in the center of the navbar

:recycle:  The NY Times logo is a link

:heavy_check_mark:   It has both, 'Subscribe' and 'Log in' elements with a `<button>` tag on the rightmost side of navbar

:heavy_check_mark:    The 'Subscribe' and 'Log in' buttons have a blue background and white letters

#### Bottom
:recycle:   It should have a 'Read comments' button and social media icons (fontawsome is recommended for the icons)

#### Additional Links
:recycle:  The section should be made of two parts: left side for links with images and right for links only

:recycle:  The left side is made of two parts: *More in Space and Cosmos* and *Editor's picks*

#### Footer
:recycle:  It includes the NYTimes logo as an embedded image in the top left corner of the footer

:recycle:  The NYTimes logo is a link

:recycle:  It includes a *go to homepage* link on the top right corner of the footer

:recycle:  The bottom part of the footer to includes links with copyright, TOC, contacts, and help


### Crucial things to make project works
:recycle:   Check page in web browser: nothing is messed up?

Full Screenshot of web page

#### Navigation bar
:recycle:   The Navbar should have a fixed position

:heavy_check_mark:     Floats or flex display for positioning items inside the navbar

#### Main Content
:heavy_check_mark:     The main content of the website should be wrapped with an HTML tag  `<main>`

:recycle:   The first image should be ~ 50% wider than the rest of the content inside `<main>`

:recycle:   All images and videos should be enclosed by a `<figure>` tag with `<figcaption>` that gives a description of the embedded image or video

:recycle:   Any sections that include text (paragraphs) should be wrapped with a `<section>` tag and include an `<aside>` tag for Editor's picks and a Sign-Up button.

#### Additional Links
:recycle:   The left side uses `<figure>` and `<figcaption>` tags followed by the title of the articles

:recycle:   `display: flex` or *float* are recommended

#### Footer
:heavy_check_mark:     The footer section is wrapped by a `<footer>` tag

:recycle:    Links should be organized using the `<ul>` tag and positioned with `display: flex` or *floats*


Keep up the good work and best of luck on the next submission. 👍

Happy Coding! :woman_technologist: :man_technologist:  :dizzy: and Have a Nice Day :smile:

Cheers,  
Bola

File: styles.css [hunk 13]
```diff
@@ -0,0 +1,480 @@
...
+}
+
+.nav-bar {
+    display: flex;
+    background-color: #fff;
+    margin-bottom: 5px;
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
```
Comment (bolah2009) [inline styles.css:56]: `SUGGESTION`

Consider adding a bottom border to give the navigation bar a visual separation from the main content of the page:
    `border-bottom: 1px solid <color>;`
Add a z-index to make the navigation bar always atop other elements:
    `z-index: <any integer value>`

#### The app currently looks like this: :point_down: 

Current App

#### Making these changes will make it look like this: :point_down: 

Prefered Look

File: styles.css [hunk 14]
```diff
@@ -0,0 +1,480 @@
...
+
+.fa-search,
+.fa-bars {
+    margin: 10px 15px;
+
+}
+
+.space-text {
+    margin: 8px 20px;
+    text-decoration: none;
```
Comment (bolah2009) [inline styles.css:71]: `SUGGESTION`

Consider adding a hover effect like shown below, this makes it look even more like the original app.

Hover effect

**Review by bolah2009 — APPROVED** [review]
# Congratulations Team! :tada:

You've done an awesome Job :muscle:

I'm impressed and proud of your progress, you should be too :clap:

##### Before you merge checkout your [html code with this validator](https://validator.w3.org/nu/#textarea) and my comments ( [this](https://github.com/onedebos/nyt-clone/pull/6#discussion_r303746226) and [this](https://github.com/onedebos/nyt-clone/pull/6#discussion_r303746886)), this will make the app look even better. 

Good luck with your next project
Keep it up :star2:

Cheers
Bola

File: styles.css (hunk 14 above)
Comment (fatymahmed) [reply styles.css:71]: Thank you for the suggestions.
we've made the changes.

**Pull request merged**